src/server/pps/cmds/cmds.go

```diff
@@ -312,6 +319,69 @@ All jobs created by a pipeline will create commits in the pipeline's repo.
 		}),
 	}
 
+	runPipeline := &cobra.Command{
+		Use:   "run-pipeline pipeline-name [-f job.json]",
+		Short: "Run a pipeline once.",
+		Long:  fmt.Sprintf("Run a pipeline once, optionally overriding some pipeline options by providing a spec.  The spec looks like this:\n%s", exampleRunPipelineSpec),
+		Run: pkgcobra.RunFixedArgs(1, func(args []string) error {
+			apiClient, err := getAPIClient(address)
+			if err != nil {
+				return err
+			}
+			pipelineName := args[0]
+			pipelineInfo, err := apiClient.InspectPipeline(context.Background(), &ppsclient.InspectPipelineRequest{
+				Pipeline: &ppsclient.Pipeline{
+					Name: pipelineName,
+				},
+			})
```

Comment:
I was thinking that we could augment the semantics of `CreateJob` a little bit to get us there. Basically make it so that you can call `CreateJob` with just `CreateJobRequest.Pipeline` set and it will go to the db to find the transform. We could probably then simplify the way we call `CreateJob` from `runPipeline` as well. Does that make sense?